modules/ui/intro/rapid.js

```diff
@@ -85,33 +86,23 @@ export function uiIntroRapid(context, reveal) {
         if (msec) { reveal(null, null, { duration: 0 }); }
         context.map().centerZoomEase(tulipLaneMid, 18.5, msec);
 
-        timeout(function() {
-            reveal(
-                'button.fb-roads-toggle',
-                t('intro.rapid.ai_roads', { rapid: icon('#iD-logo-rapid', 'pre-text') })
-            );
-
-            var button = d3_select('.fb-roads-toggle');
-
-            button.on('click.intro', function() {
-                continueTo(selectRoad);
-            });
-        }, msec + 100);
-
-        function continueTo(nextStep) {
-            context.on('enter.intro', null);
-            nextStep();
-        }
+        reveal(
+            'button.ai-features-toggle',
+            t('intro.rapid.ai_roads', { rapid: icon('#iD-logo-rapid', 'pre-text') }),
+            { buttonText: t('intro.ok'), buttonCallback: selectRoad });    
     }
 
 
     function selectRoad() {
+        var drawAiFeatures = context.layers().layer('ai-features');
+        drawAiFeatures.enabled(!drawAiFeatures.enabled());
```

Comment:
ha, yet another case where the ouroborous eats itself. Yes, that's exactly what this code is supposed to be doing- enabling the roads layer so that the user can select an AI road.